services/Mumble/MumbleUser.ts

```diff
@@ -2,27 +2,65 @@ import config from '@/config';
 import { MumbleService } from '@/services/Mumble/index';
 import { EApiMethods, EEndpointsBackend } from '@/utils/enums/general.enum';
 import { generateBoundary } from '@/utils/helpers/generateBoundary';
-import { IMumbleFollowersApiResponse } from '@/utils/interfaces/mumbleFollowers.interface';
+import {
+  IMumbleFollowers,
+  IMumbleFollowersApiResponse,
+} from '@/utils/interfaces/mumbleFollowers.interface';
+import { IMumbleUser } from '@/utils/interfaces/mumbleUsers.interface';
 
 export interface IUploadAvatarParams {
   token: string;
   image: File;
 }
 
-export interface IUserMumble {
-  id: string;
-  username: string;
-  avatarUrl: string;
-}
-
 export class MumbleUserService extends MumbleService {
+  async getAllUsers({
+    token,
+    users,
+    url,
+    ttl,
+  }: {
+    token: string;
```

Comment:
Perhaps creating an interface for getAllUsers might make sense to keep it consistent through the project.